v8-cmake fails to build torque

Been trying to build torque, however it always fails with `The system cannot execute the specified program`.

I tried enabling windows long paths and older versions (v.8.9 and v.8.8), still fails.

```
...
C:/src/out/dev/vendor/v8-cmake/torque-generated/objects-printer-tq.cc C:/src/out/dev/vendor/v8-cmake/torque-generated/objects-printer.cc 
  vendor\v8-cmake\CMakeFiles\aggregate-error-tq-csa.cc-851e72b.bat 617396520723a9cb
  The system cannot execute the specified program.
  
  Batch file failed at line 4 with errorcode 1
  
  ninja: build stopped: subcommand failed.
```

> Been trying to build torque, however it always fails with `The system cannot execute the specified program`.
> 
> I tried enabling windows long paths and older versions (v.8.9 and v.8.8), still fails.
> 
> ```
> ...
> C:/src/out/dev/vendor/v8-cmake/torque-generated/objects-printer-tq.cc C:/src/out/dev/vendor/v8-cmake/torque-generated/objects-printer.cc 
>   vendor\v8-cmake\CMakeFiles\aggregate-error-tq-csa.cc-851e72b.bat 617396520723a9cb
>   The system cannot execute the specified program.
>   
>   Batch file failed at line 4 with errorcode 1
>   
>   ninja: build stopped: subcommand failed.
> ```

Don't use Ninja on MSVC, it's buggy. default should be fine.

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -999,8 +999,11 @@ target_include_directories(v8_torque_generated
     ${PROJECT_SOURCE_DIR}/v8
 )
 
-# The torque-outputs and torque_outputs variables can have too many characters on Windows to be executed on the command line, so we have to write it to a temporary file and execute that 
-file(WRITE "${PROJECT_BINARY_DIR}/touch_torque_outputs.cmake" "file(TOUCH ${torque-outputs};${torque_outputs})")
+# The torque-outputs and torque_outputs variables can have too many characters
+# on Windows to be executed on the command line, so we have to write it to a
+# temporary file and execute that 
+file(WRITE "${PROJECT_BINARY_DIR}/touch_torque_outputs.cmake"
+           "file(TOUCH ${torque-outputs};${torque_outputs})")
 
 add_custom_command(
   COMMAND

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -999,6 +999,9 @@ target_include_directories(v8_torque_generated
     ${PROJECT_SOURCE_DIR}/v8
 )
 
+# The torque-outputs and torque_outputs variables can have too many characters on Windows to be executed on the command line, so we have to write it to a temporary file and execute that 
+file(WRITE "${PROJECT_BINARY_DIR}/touch_torque_outputs.cmake" "file(TOUCH ${torque-outputs};${torque_outputs})")
+
 add_custom_command(
   COMMAND
     torque
@@ -1006,7 +1009,7 @@ add_custom_command(
     -v8-root ${PROJECT_SOURCE_DIR}/v8
     ${torque_files}
   COMMAND
-    ${CMAKE_COMMAND} -E touch ${torque-outputs} ${torque_outputs}
+    ${CMAKE_COMMAND} -P ${PROJECT_BINARY_DIR}/touch_torque_outputs.cmake
   DEPENDS
     torque
     ${torque_dirs}

